src/Sylius/Bundle/ShopBundle/Resources/views/Product/Show/_associations.html.twig

```diff
@@ -1,2 +1,9 @@
-<h4 class="ui horizontal section divider header">{{ 'sylius.ui.you_may_also_like'|trans }}</h4>
-{{ render(url('sylius_shop_partial_product_index_latest', {'count': 4, 'template': '@SyliusShop/Product/_simpleList.html.twig'})) }}
+{% for association in product.associations %}
+    <div>
+        <h4 class="ui horizontal section divider header">{{ association.type.name }}</h4>
+        {{ render(url('sylius_shop_partial_associated_products', {'id': association.id, 'template': '@SyliusShop/Product/_associations.html.twig'})) }}
```

Comment:
Why do we need to call controller? We have all associations already loaded in the template, we can simply replace this line with `{% include '@SyliusShop/Product/_simpleList.html.twig' with {'products': association.associatedObjects} %}`, can't we?